### PR TITLE
Update assigning-issues-and-pull-requests-to-other-github-users.md

### PR DESCRIPTION
**Pull request by sammachin**
### Why:

Closes [issue link](https://github.com/github/feedback/discussions/7150)

### What's being changed:

Clarify that multiple assignees isn't available for private repos on a free account

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@sammachin 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Sou9pp — COMMENTED** [review]
[Soon040106@kakao.com](Git-2.35.1.2-32-bit.exe)

File: content/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ shortTitle: Assign issues & PRs
 
 ## About issue and pull request assignees
 
-You can assign up to 10 people to each issue or pull request, including yourself, anyone who has commented on the issue or pull request, anyone with write permissions to the repository, and organization members with read permissions to the repository. For more information, see "[Access permissions on {% data variables.product.prodname_dotcom %}](/articles/access-permissions-on-github)."
+You can assign multiple people to each issue or pull request, including yourself, anyone who has commented on the issue or pull request, anyone with write permissions to the repository, and organization members with read permissions to the repository. For more information, see "[Access permissions on {% data variables.product.prodname_dotcom %}](/articles/access-permissions-on-github)."
+
+All Public repositories and private repositories on a paid account can have up to 10 people assigned. Private repoistories on the Free plan are limited to one person per issue.
```
Comment (hubwriter) [inline content/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users.md:24]:
```suggestion
Issues and pull requests in public repositories, and in private repositories for a paid account, can have up to 10 people assigned. Private repositories on the free plan are limited to one person per issue or pull request.
```

**Review by hubwriter — APPROVED** [review]

**Comment by hubwriter** [comment]
@sammachin - Many thanks for contributing this clarification to the documentation.

I'll get this merged/published. 🚀

**Pull request merged**